bin/index.js

```diff
@@ -12,10 +12,12 @@ const {
   clientConfig: clientConfigPath,
   devConfig: devConfigPath,
   port = 8080,
+  testOpts,
   _
 } = require('yargs').argv
 
 const action = _[0]
+const extras = _.slice(1).join(' ') || undefined
```

Comment:
Why default this to `undefined`?
If there's nothing in that array the result is the empty string `''`, what's the matter with that?